### ISSUE TITLE
What does this mean?! HELP!

### ISSUE THREAD

**Issue by Dani141990** [ZaneYork/SMAPI-Android-Installer#448]
[11:30:35 ERROR SMAPI] Thread failed:
Java.IO.FileNotFoundException: Content/XACT/music.xwb 

It keeps crashing the game and there seems no clear pattern to it. I've taken all the mods out and still get this. It just pops up randomly. I can play for days and days in game and then any random moment this happens!

**Comment by PurplePanda395** [comment]
Did you ever find a way to fix it? I'm having the same problem

**Comment by Dani141990** [comment]
Yeah. A bit extreme but it stopped the crashes completely. I used MT manager to edit the apk files and deleted the music.xwb file. Lost some sounds and music but it stopped crashing and it didn't keep me from playing properly. I actually beat the game like that.

**Comment by PurplePanda395** [comment]
Awesome! I tried it, and it just froze, but I started up stardew without SMAPI and it did the same thing so I think this is just my phone now lol thanks though!!
